lib/ansible/modules/user.py

```diff
@@ -250,6 +250,18 @@
             - Supported on Linux only.
         type: int
         version_added: "2.11"
+    subuid:
+        description:
+            - UID range allocated for use by newuidmap.
+            - Supported on Linux only.
+            - Currently only C(subuid=present) and C(subuid=absent) are supported.
```

Comment:
```suggestion
            - Currently only I(subuid=present) and I(subuid=absent) are supported.
```
See https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation

(Same below.)